### DISCUSSION TITLE
OpenAI釋出新AI，輸入文字就能快速產出3D模型

### DISCUSSION THREAD

**Discussion by RogelioKG** [1101-GS3073-C/Python-Community-GS3073-C#582] [General]
`#3D模型`   `#生成式AI`   `#以文生圖`

Content:
https://www.ithome.com.tw/news/154940

Reflections / Thoughts:
相較於以往 GPU 可能要花上一小時才能將 3D 模型渲染好 (例如光照貼圖的烘焙就要花費大量時間)。Point-E 的模型點雲技術的優點在於處理速度極快，只需要1~2分鐘就能快速渲染一個 3D 模型，儘管模型品質無法達到如電影或遊戲中的商業 3D 渲染，然而我們可以將 3D 模型導入其他 3D 應用程式再進行修改，這大大減少模型建立所耗費的時間。

Reference:
https://chinese.engadget.com/openai-releases-point-e-dall-e-3-d-text-modeling-130058200.html

**Comment by skyling0299** [comment]
3D 生成物是以點狀雲組成，解析度不怎麼樣，不過其實這只需要一個 GPU 核心和幾分鐘就能完成，相較 Google DreamFusion 需要多核 GPU 機器和數小時的運算，這樣的成果也算不錯了吧。

**Comment by manipulateed** [comment]
這比較像是走捷徑的做法，但在這個講究效率的時代，能快速生產基底，再去仔細加工，細節雕刻，而不用從頭到尾，從無到有，才是趨勢也是更有競爭力的做法吧。
